File: src/main/kotlin/com/lcaohoanq/sp/repositories/NotificationRepository.java
package com.lcaohoanq.sp.repositories;

import com.lcaohoanq.sp.entities.NotificationEntity;
import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;

public interface NotificationRepository extends JpaRepository<NotificationEntity, Long> {

    List<NotificationEntity> findByUserEmail(String email);
    List<NotificationEntity> findByUserEmailOrderByCreatedAtDesc(String email);

}

File: src/main/kotlin/com/lcaohoanq/sp/repositories/NotificationRepository.kt
package com.lcaohoanq.sp.repositories

import com.lcaohoanq.sp.entities.NotificationEntity
import org.springframework.data.jpa.repository.JpaRepository

interface NotificationRepository : JpaRepository<NotificationEntity, Long> {
    fun findByUserEmail(email: String): MutableList<NotificationEntity>
    fun findByUserEmailOrderByCreatedAtDesc(email: String): MutableList<NotificationEntity>
}
